lumigator/python/mzai/sdk/tests/int/test_scenarios.py

```diff
@@ -10,9 +11,11 @@
 from lumigator_schemas.datasets import DatasetFormat
 from lumigator_schemas.jobs import JobEvalCreate, JobType
 
-'''
+"""
 Test the healthcheck endpoint.
```

Comment:
Since we're already modifying this code, I think we can remove this comment, wdyt?

Reply:
Ok